lib/task.js

```diff
@@ -149,6 +150,22 @@ function factory(
             });
     };
 
+    Task.prototype.getPath = function(obj, path) {
```

Comment:
This function looks like directly copy from [catalog-searcher](https://github.com/RackHD/on-tasks/blob/master/lib/utils/job-utils/catalog-searcher.js#L20).  Actually, the lodash [_.get](https://lodash.com/docs#get) API should fulfill the case here, so I think we may not need to implement this ourselves.